### PR TITLE
clean up code, sort, add comments

### PR DESCRIPTION
**Pull request by danrubel**
@devoncarew

### CODE REVIEW

File: dart/tool/java/src_gen_java.dart [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// A library to generate Java source code.
+/// See [JavaGenerator].
```
Comment (devoncarew) [inline dart/tool/java/src_gen_java.dart]: Markdown will emit these as one line. I would either add a blank line between the two lines if that is your intention, or just write this comment as one line.

**Comment by devoncarew** [comment]
lgtm!

**Comment by danrubel** [comment]
Merged

File: dart/tool/java/src_gen_java.dart (hunk 1 above)
Comment (danrubel) [inline dart/tool/java/src_gen_java.dart]: Good point. Done.

**Pull request closed without merging**